src/_data/mde.yml

```diff
@@ -10,6 +10,16 @@ extensions:
   -
     name: Amazon Sales Channel
     versions:
+      -
+        name: 4.3.0
+        support:
+          2.3.0: compatible
+          2.3.1: compatible
+          2.3.2: compatible
+          2.3.3: compatible
+          2.3.4: compatible
+          2.3.5-p1: compatible
+          2.4.0: compatible
```

Comment:
As the next releases are due out in just over a week, night it not be prudent to add - 
2.4.1
2.4.0-p1
2.3.6

Reply:
they are not released yet.  the table will be updated when those are released on the 15th

Reply:
Okay - just thought it should be checked.

Reply:
thanks for the question.  i'll update the 2.4.1 branch with the asc 4.3.0 today